test/google_search_data_viewer_web/controllers/session_controller_test.exs

```diff
@@ -0,0 +1,56 @@
+defmodule GoogleSearchDataViewerWeb.SessionControllerTest do
+  use GoogleSearchDataViewerWeb.ConnCase
+  import GoogleSearchDataViewer.AccountsFixtures
+
+  @valid_email "test@gmail.com"
+  @valid_password "aValidPasswordEntered"
+
+  describe "POST create/2" do
+    test "signs a user in given correct email and password", %{conn: conn} do
```

Comment:
We prefer to bring the `given` part to the first place. Reference: https://nimblehq.co/compass/development/code-conventions/elixir/ex-unit/#formatting

```suggestion
    test "given correct email and password, signs a user in", %{conn: conn} do
```

Reply:
Thanks, changed.